integration-tests/tests/src/tests/sync/user.ts

```diff
@@ -327,6 +327,25 @@ describe.skipIf(environment.missingServer, "User", () => {
         expect(didFail).to.be.true;
       });
 
+      it("can switch user", async function (this: Mocha.Context & AppContext & RealmContext) {
+        expect(this.app.currentUser, "No users").to.be.null;
+
+        const user1 = await registerAndLogInEmailUser(this.app);
+        const user2 = await registerAndLogInEmailUser(this.app);
+
+        expect(this.app.currentUser?.id).to.equal(user2.id);
+        expect(Object.keys(this.app.allUsers)).to.have.lengthOf(2);
```

Comment:
Wonder if we have a cleanup issue here.  The user count is `7` instead of `2` in the tests.